src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/validator/AppointmentValidator.kt

```diff
@@ -63,8 +64,11 @@ class AppointmentValidator {
     behaviourDescription: String?,
     errors: MutableList<FieldError>
   ) {
-    if (appointmentTime.isAfter(OffsetDateTime.now()))
+    // if the appointment occurred today or on a date in the future, no attendance validation is required
+    if (appointmentTime.isAfter(OffsetDateTime.now().with(LocalTime.MIN))) {
```

Comment:
Just musing: later perhaps we could also introduce a return value as currently both error and non-error cases use an empty `return`, making understanding the side-effect a little bit more effort to follow

Reply:
on the regression thing - i'm not super bothered if this stays around. worst case is the SP is allowed to enter a session that happens earlier today and they're not forced to enter feedback right away. they are still contractually obliged to enter feedback on the same day, it's just a few more clicks.

but yeh i take your point that having a reminder would be beneficial. i think just tracking in trello is sufficient though.

Reply:
as a side note though - it sucks no unit tests started failing when i made this change.

Reply:
added unit test